samples/led/src/main.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <npmx_driver.h>
+#include <npmx_led.h>
+
+#define LOG_MODULE_NAME pmic_led
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+/**
+ * @brief Set LEDs status based on lower 3 bits of status variable.
```

Comment:
Do we want to stick to the header files' standards? Like: function for setting...

Reply:
@masz-nordic what do you think?

For me, we can leave it as it is.